From 31cfc47577ff58c90cfe11de403810e4064bdcb2 Mon Sep 17 00:00:00 2001
From: Kyle Gospodnetich <me@kylegospodneti.ch>
Date: Wed, 29 Jul 2026 07:02:49 -0700
Subject: [PATCH 80/84] Merge pull request #85 from
 OpenGamingCollective/oxp-led-rename

[NOT-FOR-UPSTREAM] HID: hid-oxp: Report RGB LED under the Legion Go name
---
 drivers/hid/hid-oxp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hid/hid-oxp.c b/drivers/hid/hid-oxp.c
index 20a54f337220..b7427ebf6d1d 100644
--- a/drivers/hid/hid-oxp.c
+++ b/drivers/hid/hid-oxp.c
@@ -1386,7 +1386,7 @@ static struct mc_subled oxp_rgb_subled_info[] = {
 
 static struct led_classdev_mc oxp_cdev_rgb = {
 	.led_cdev = {
-		.name = "oxp:rgb:joystick_rings",
+		.name = "go:rgb:joystick_rings",
 		.color = LED_COLOR_ID_RGB,
 		.brightness = 0x64,
 		.max_brightness = 0x64,
-- 
2.55.0

